testsuite/features/step_definitions/command_steps.rb

```diff
@@ -179,6 +179,17 @@
   $command_output, _code = get_target('server').run(command)
 end
 
+When(/^I use spacewalk-common-channel to add all "([^"]*)" channels with arch "([^"]*)"$/) do |channel, architecture|
+  channels_to_synchronize = CHANNEL_TO_SYNC_BY_OS_PRODUCT_VERSION.dig(product, "#{channel}-#{architecture}")
+  raise ScriptError, "Synchronization error, version type #{channel}-#{architecture} in #{product} product not found" if channels_to_synchronize.nil?
```

Comment:
speaking of aligning code between implementations, this guard clause is missing from the other similar step definitions :disappointed: , if you have time feel free to add it there

Reply:
I have found two additional occurrences, one I adjusted to the relevant `next if` statement, and the second now raises exception if a given path does not exist.

Reply:
thanks